composer.json

```diff
@@ -48,7 +48,8 @@
         "mongodb/mongodb": "^1.0",
         "predis/predis": "^1.0",
         "squizlabs/php_codesniffer": "~2.0",
-        "vlucas/phpdotenv": "^2.4.0"
+        "vlucas/phpdotenv": "^2.4.0",
+        "symfony/process": "^3.3"
```

Comment:
Yes, thank you for pointing. Actually we already have this dependency by facebook/webdriver so it should not add anything new to vendor or phar, but should be specified explicitly